Index: lispd-tun/lispd/Makefile
===================================================================
--- lispd-tun.orig/lispd/Makefile	2012-12-19 15:48:32.000000000 +0100
+++ lispd-tun/lispd/Makefile	2012-12-19 16:01:16.942206926 +0100
@@ -16,11 +16,11 @@
 endif
 GENGETOPT	= gengetopt
 
-CFLAGS		+= -Wall -g -DDEBUG=3 -DTESTLOCALEID
-#CFLAGS		+= -Wall -g -DDEBUG=3 -DTESTLOCALEID -DOPENWRT
+#CFLAGS		+= -Wall -g -DDEBUG=3 -DTESTLOCALEID
+CFLAGS		+= -Wall -g -DDEBUG=3 -DTESTLOCALEID -DOPENWRT
 
-LIBS		= -lconfuse -lssl -lcrypto -lrt -lm
-#LIBS		= -lconfuse -lssl -lcrypto -lrt -lm -luci
+#LIBS		= -lconfuse -lssl -lcrypto -lrt -lm
+LIBS		= -lconfuse -lssl -lcrypto -lrt -lm -luci
 INC		= lispd.h
 MAKEFILE	= Makefile
 OBJS		= cmdline.o lispd.o lispd_config.o lispd_log.o	\
